COD60275.TXT

```diff
@@ -0,0 +1,260 @@
+OBJECT Codeunit 60275 Xml Dom Mgt.
```

Comment:
Note that we already have codeunit 6224 "XML DOM Management" which seems very similar to this codeunit. Perhaps we are missing some functions in codeunit 6224?